src/lib.rs

```diff
@@ -47,7 +46,7 @@ use sound::RollbackSoundManager;
 use windows::core::PCWSTR;
 use windows::Win32::Foundation::HANDLE;
 use windows::Win32::System::LibraryLoader::GetModuleFileNameW;
-use windows::Win32::System::Memory::{HEAP_FLAGS, HEAP_ZERO_MEMORY};
+use windows::Win32::System::Memory::{HEAP_FLAGS};
```

Comment:
`HEAP_ZERO_MEMORY` will be used if `fillfree` feature is enabled.
https://github.com/Hagb/giuroll-hagb/blob/48224ac471dafbed75489ba14d914bb5819f4620/src/lib.rs#L2242-L2245
To avoid the warning, it can be imported like
```rust
use windows::Win32::System::Memory::HEAP_FLAGS;
#[cfg(feature = "fillfree")]
use windows::Win32::System::Memory::HEAP_ZERO_MEMORY;
```
or used as `windows::Win32::System::Memory::HEAP_ZERO_MEMORY` directly on https://github.com/Hagb/giuroll-hagb/blob/48224ac471dafbed75489ba14d914bb5819f4620/src/lib.rs#L2243

Reply:
https://github.com/Hagb/giuroll-hagb/actions/runs/11886090480